codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/AwsQuery.kt

```diff
@@ -46,18 +49,24 @@ class AwsQueryFactory : ProtocolGeneratorFactory<HttpBoundProtocolGenerator> {
     }
 }
 
+private val awsQueryHttpTrait = HttpTrait.builder()
```

Comment:
Nit: I think this would be better in the constructor of AwsQueryBindingResolver so that it isn't constructed on startup if the codegen isn't generating an AWS Query protocol.